### PR TITLE
New Sector LCCC-H_CTR added

### PR DESCRIPTION
**Pull request by nakoale**

### CODE REVIEW

**Comment by nakoale** [comment]
UKFF renamed URFF Simferopol, Russia. UKFV_CTR renamed URFV_CTR

**Comment by NelisV** [comment]
LCCC-H overlaps with the already existing LCCC-S. I would also advice against changing the ownership of the simferopol sector. In AIRAC2010 it belongs to Ukraine.

**Comment by Adrian2k** [comment]
URFV_CTR is not allowed on VATSIM as VATSIM does not involve itself in political conflicts.
It is stated both on VATSIM Ukraine and VATSIM Russia's website that it is UKFV_CTR.

URFV is not recognized by anyone else than Russia. A simple google search will find your answers.

**Comment by Adrian2k** [comment]
URFV has also been discussed here: https://github.com/vatsimnetwork/vatspy-data-project/pull/64

File: VATSpy.dat
Comment (prryplatypus) [inline VATSpy.dat:12998]: Intended as per #77 and #64. Please undo this change.

File: VATSpy.dat
Comment (prryplatypus) [inline VATSpy.dat:18001]: Intended as per #77 and #64. Please undo this change.

File: FIRBoundaries.dat
Comment (prryplatypus) [inline FIRBoundaries.dat:24122]: Intended as per #77 and #64. Please undo this change.

File: FIRBoundaries.dat
Comment (prryplatypus) [inline FIRBoundaries.dat:9514]: This sector overlaps with `LCCC-S`. Is this intended? Also try keeping them ordered alphabetically, please.

File: VATSpy.dat
Comment (prryplatypus) [inline VATSpy.dat:17793]: Please move this between `LCCC-E` and `LCCC-S` to keep them alphabetically sorted.

File: FIRBoundaries.dat
Comment (nakoale) [reply FIRBoundaries.dat:9514]: Yes the change is intended cause LCCC-H is a subsector of LCCC-S. Will move it alphabetically.

File: FIRBoundaries.dat
Comment (nakoale) [reply FIRBoundaries.dat:24122]: The reason i did this change is because next days VATRUS has an event focused on URFF and i was thinking that we need to change it to let pilots know that this area is online that day. If we leave it UKFF noone will know and unfortunately from the moment this icao does not exist , no metar is shown. what do you think guys?

**Comment by nakoale** [comment]
LCCC-H moved alphabetically. Waiting for reply regarding URFV ,URFF

File: FIRBoundaries.dat
Comment (prryplatypus) [reply FIRBoundaries.dat:24122]: As stated in their websites and the issues mentioned above, this should indeed be UKFV.

P.D.: How does an FIR have a METAR?

File: FIRBoundaries.dat
Comment (NelisV) [reply FIRBoundaries.dat]: Have some doubts about adding this split, in the stats database i cannot see more than 1 minute of use for LCCC-H, and no more than 6 minutes total for LCCC-C. it's not for me to decide, but is the extra 'clutter' really necessary? Is Cyprus going to use this split? 
image

**Comment by NelisV** [comment]
Have some doubts about adding this split, in the stats database i cannot see more than 1 minute of use for LCCC-H, and no more than 6 minutes total for LCCC-C. it's not for me to decide, but is the extra 'clutter' really necessary? Is Cyprus going to use this split? Looking at the hours, i would go for merging, instead of splitting further :).

image

File: FIRBoundaries.dat
Comment (nakoale) [reply FIRBoundaries.dat:24122]: im talking about the airport of Simferopol UKFF/URFF not the FIR. from the moment this area in real life is under the russian ACC then all the icao codes have been renamed to URXX. So the metar will show under URFF only. So my proposal is to leave UKFV CTR as it is and at least rename the airport to URFF? how does it sound?

**Comment by nakoale** [comment]
as for the LCCC-H it is a real sector that opens in heavy workload especially during heavy traffic going to Tel Aviv transit Greece. So we build it a couple of days ago. On Vatsim such events may occur so we suggest we put it on vatspy

File: FIRBoundaries.dat
Comment (NelisV) [reply FIRBoundaries.dat:24122]: Only in the russian AIP have the names been ammended to URFF, all other authorities like ICAO, IATA, Navigraph (but also vatsim, vatrus and vatukraine) still use UKFF.

Comment (nakoale) [reply FIRBoundaries.dat:24122]: Ok. should i change it back to default?

Comment (prryplatypus) [reply FIRBoundaries.dat:24122]: Please do! 😛

**Comment by meltinglava** [comment]
Just a request that thise commits gets squashed before they get merged.

**Review by prryplatypus — APPROVED** [review]

**Pull request merged**
